src/proxy.ts

```diff
@@ -11,6 +12,12 @@ export function initializeProxy(): void {
     const MAJOR_NODEJS_VERSION = parseInt(process.version.slice(1).split('.')[0], 10);
 
     if (MAJOR_NODEJS_VERSION >= 10) {
+      // See: https://github.com/electron/get/pull/214
```

Comment:
This should link to the specific discussion.

```suggestion
      // See: https://github.com/electron/get/pull/214#discussion_r798845713
```